netci.groovy

```diff
@@ -117,13 +117,13 @@ def CreateStyleCheckTasks = { taskString, taskName, checkName ->
         def jobName = Utilities.getFullJobName(project, taskName, isPR)
 
         def newJob = job(jobName) {
-            label('ubuntu')
             steps {
                 shell(taskString)
             }
         }
 
         Utilities.simpleInnerLoopJobSetup(newJob, project, isPR, checkName)
+        Utilities.setMachineAffinity(newJob, 'Ubuntu14.04', 'latest-or-auto')
```

Comment:
Created as a need arises for the label.  It takes around 3-5 minutes for Linux VMs.  VMs will stick around for about an hour if idle before being deallocated, so there is a great change that during the day there will be no wait.